packages/api-specification/interface/screen.ts

```diff
@@ -0,0 +1,36 @@
+declare namespace ssf {
+  export abstract class Screen {
+    static getDisplays(): Promise<Array<Display>>;
+  }
+
+  /**
+   * Information about the users display. Note that the Browser API does not support multiple
+   * displays, so it assumes the display the browser is running in is the only display.
+   */
+  export interface Display {
+    /**
+     * Unique Id of the display.
+     */
+    id: string;
```

Comment:
Honestly i'm not 100% sure, but its something that both Electron and OpenFin return. I think if we add more display methods it would make more sense (i.e. checking the display returned from `getPrimaryDisplay()` or `getFocusedDisplay()` and checking against the list of all displays).